.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,8 +35,8 @@ Why is this change required? What problem does it solve?
 If it fixes an open issue, please link to the issue here.
 
 If you don't have an issue, we'd recommend starting with one first so the PR
-can focus on the implementation (unless its an obvious bug or documentation fix
-that will have little conversation).
+can focus on the implementation (unless it is an obvious bug or documentation fix
+that will have a little conversation).
```

Comment:
The intent here is: "please open an issue for this PR if one doesn't already exist, unless this is an obvious bug fix that requires little to no conversation."

So we can remove the `a` here, and keep it as either:

```suggestion
that will have little conversation).
```

or if we want to be verbose:

```suggestion
that will have little to no conversation).
```